.github/workflows/pr_check_workflow.yml

```diff
@@ -3,11 +3,12 @@
 
 name: Build and test
 
+# trigger on every commit push and PR for all branches except feature branches
 on:
   push:
-    branches: [ main ]
```

Comment:
Are there any issues with line 67, 164, and 192 and none `main` branches?

(tbh these values can probably be pulled up higher to a variable)

Reply:
Ah yes I see what you mean. The node and chromedriver version are different for the other branches. Good catch. Looks like for node we can use the `.nvmrc` file via the [`node-version-file` property](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file). For chromedriver we don't have a choice right because GitHub only has the latest version?

Reply:
Awesome!